src/libexpr/eval-settings.hh

```diff
@@ -19,9 +19,9 @@ struct EvalSettings : Config
         Enable built-in functions that allow executing native code.
 
         In particular, this adds:
-        - `builtins.importNative` *path*
-        
-          Load a dynamic shared object (DSO) at *path* which exposes a function pointer to a procedure that initialises a Nix language value, and return that value.
+        - `builtins.importNative` *path* *symbol*
+          Runs function with *symbol* from a dynamic shared object (DSO) at *path*.
+          This may be used to add new builtins to the Nix language.
```

Comment:
```suggestion
          This may be used to add new primitive functions to the Nix language, that may not be possible to express within the language itself.
          Calls to `importNative` do not modify `builtins`.
```